dali/python/nvidia/dali/plugin/pytorch/torch_utils.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Also, [I still think we need a test for error propagation](https://github.com/NVIDIA/DALI/pull/5726#issuecomment-2507466525)